src/styles/blocks/card.css

```diff
@@ -0,0 +1,92 @@
+.card {
+  padding: 32px 16px 16px;
```

Comment:
Consider adding 'box-sizing: border-box;' to the .card class to include padding and border in the element's total width.